config/custom-environment-variables.json

```diff
@@ -70,5 +70,6 @@
   "chain_id": "SDC_CHAIN_ID",
   "address_prefix": "SDC_ADDRESS_PREFIX",
   "conveyor_posting_wif": "CONVEYOR_POSTING_WIF",
-  "conveyor_username": "CONVEYOR_USERNAME"
+  "conveyor_username": "CONVEYOR_USERNAME",
+  "pinned_posts_url": "CONVEYOR_PINNED_POSTS_URL"
```

Comment:
Could we remove `CONVEYOR` from the env var since it doesn't seem to rely on the conveyor service at all?

Reply:
agree, let's just call it PINNED_POSTS_URL - sg @rjungemann ?

Reply:
Oops I changed this and I think I rolled it back. Sorry